bin/_tag.sh

```diff
@@ -24,7 +24,7 @@ head_root_tag() {
     if clean_head ; then
         clean_head_root_tag
     else
-        echo "dev-$(git_sha_head)-$USER"
+        echo "dev-$(git_sha_head)-$(echo $USER | sed "s/[^[:alpha:].-]//g")"
```

Comment:
This might be cleaner to split onto it's own line:
```
name="$(...)"
echo echo "dev-$(git_sha_head)-$name"
```

Reply:
What do you think of using `[:alnum:]` to allow numbers as well?

Reply:
Yes, valid points. I'll incorporate your suggestions. Thanks.